### DISCUSSION TITLE
[Feature Request] Drag to reorder rows

### DISCUSSION THREAD

**Discussion by autohub7** [heroui-inc/heroui#1243] [Feature requests]
### Is your feature request related to a problem? Please describe.

I would like to have a function that permits the dragging and rearranging of rows within a table.

### Describe the solution you'd like

I expected a table allows reordering rows by drag&drop

User drags a single row to rearrange row 
or  
User drags selected rows to rearrange row 

### Describe alternatives you've considered

https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/draganddrop
https://ant.design/components/table

**Comment by truediogo** [comment]
up

**Comment by le0zh0u** [comment]
I need Drag and Drop to reorder items in ListBox, upupup

**Comment by ziyuang** [comment]
Doesn't React Aria's `ListBox` support drag-and-drop already, if NextUI is built on top of it?

https://react-spectrum.adobe.com/react-aria/ListBox.html#features

**Comment by dhananjaykumar880** [comment]
When can we expect drag drop in listbox and table?
